scripts/run-integration-tests.js

```diff
@@ -3,8 +3,12 @@
 const helpers = require('yeoman-test');
```

Comment:
I re-worked this file a lot, we now make sure that the `version`, `ci`, `install` and `prepublishOnly` scripts will work by using the various commands. In the future we should probably add `audit` here as well.